demos/python_demos/forward_tacotron_wavernn_demo/models/mel2wave_ie.py

```diff
@@ -0,0 +1,218 @@
+import os.path as osp
+import logging as log
+import time
+
+from openvino.inference_engine import IECore
+import numpy as np
+
+from utils.wav_processing import *
+
+
+class Mel2WaveInference():
+    def __init__(self):
+        self.ie = IECore()
+
+    def load_network(self, model_xml, batch_sizes=None):
```

Comment:
This function doesn't confirm own name. It should return only `CNNNetwork` object, not both. It will be much better if you split it into two functions, e.g. `load_network` and `load_network_to_device`/`create_exec_network`/`load_exec_network`.